### ISSUE TITLE
Remove comments after fix

### ISSUE THREAD

**Issue by Djaler** [gabrie-allaigre/sonar-gitlab-plugin#161]
It would be cool if the plugin could delete comments, when these problems were fixed

**Comment by gabrie-allaigre** [comment]
Hi,
Comments if only in commit, if you fixe problems is a new commit.

**Comment by Djaler** [comment]
Yeah, but I think it should delete comments in this case too. Because in this MR there are no problems now.
